### DISCUSSION TITLE
Commercial discards

### DISCUSSION THREAD

**Discussion by aliwhitman** [shcaba/Sebastes_melanops_OR#17] [General]
Okay, just getting some documentation going here. It appears that the previous assessment used WCGOP to estimate commercial discards. From the document, it looks like they lumped estimated coastwide discards from 2002-2013 and estimated a coastwide discard rate of 1.58% by comparing total discards to total landings. This rate was added to the total commercial removals.  They note that this potentially overestimated discards, but I'm a little unclear how that could be from the documentation. 

I think the easiest course of action would be to just update the discard rate from the previous assessment and tack that on to the commercial fleet landings for each fleet, following the previous assessment's methodology but with updated data. This is assuming that WCGOP data only really have data for the hook and line fleet, so fleet-specific discard rates would not be possible.  

If there are other ideas, I'm open! Jason, if you could make that request for the WCGOP data to Chantel, that would be helpful to get us going on this one.

**Comment by aliwhitman** [comment]
Discussion at meeting today: 

Recreational discards are all set. 

Use GEMM for non-trawl commercial discard rates - Ali will look at excel file to toggle gear types and get the most appropriate rate.  Could be applied backwards in time (WCGOP only goes back to 2004).

Looked at Sampson EDCP analysis (in google folder) for "historical" (1990s) trawl discard rates - they were 0.0% so Ali will provide this report as justification for no discard mortality to add on to trawl landings.

**Comment by shcaba** [comment]
I met with Kayleigh and she showed me how to properly get the Oregon black rockfish discards.
If looking at the GEMM in an excel, you can produce a pivot table with filters groupings (Black/blue/deacon rockfish (Oregon)), sector (get rid of the Oregon recreational sector), and species (Black Rockfish). This will produce the black rockfish-specific discards for Oregon.

**Comment by aaronmberger-nwfsc** [comment]
Sounds like we have a good path forward. For completeness, here is an excerpt explaining what was done for the 2017 Blue/Deacon assessment:

"The amount of discarded BDR relative to retained BDR was estimated by the Groundfish Expanded Mortality Multiyear (GEMM) report. Discard ratios were available from 2002 to 2015 for the nearshore fixed-gear fishery (in waters < 50 fathoms). Mortality rates associated with discarded BDR are specified by depth bins following the approved levels specified by the Pacific Fisheries Management Council – Groundfish Management Team (see Somers et al. 2016). The average dead discard rate for BDR was 24.7% for the management area north of 40°10̍north latitude (Figure 8). This value was used to calculate total discarded catch by multiplying 0.247 by annual estimates of commercial landings over the time series (1892-2016)."

The cited figure is pasted here:
![image](https://user-images.githubusercontent.com/7946234/229011268-36ef9455-b2db-461e-9d77-89509fb2b9b2.png)

**Comment by shcaba** [comment]
The harmonic mean discard rate for years 2004-2022 is 0.008.
[image: image.png]
While variable, it is all very low. We could use this to go backwards to
add the low amount of discards for commercial fisheries, though I guess
we'd need to break this down by trawl and non-trawl commercial discards.

On Thu, Mar 30, 2023 at 7:56 PM Aaron Berger ***@***.***>
wrote:

> Sounds like we have a good path forward. For completeness, here is an
> excerpt explaining what was done for the 2017 Blue/Deacon assessment:
>
> "The amount of discarded BDR relative to retained BDR was estimated by the
> Groundfish Expanded Mortality Multiyear (GEMM) report. Discard ratios were
> available from 2002 to 2015 for the nearshore fixed-gear fishery (in waters
> < 50 fathoms). Mortality rates associated with discarded BDR are specified
> by depth bins following the approved levels specified by the Pacific
> Fisheries Management Council – Groundfish Management Team (see Somers et
> al. 2016). The average dead discard rate for BDR was 24.7% for the
> management area north of 40°10̍north latitude (Figure 8). This value was
> used to calculate total discarded catch by multiplying 0.247 by annual
> estimates of commercial landings over the time series (1892-2016)."
>
> The cited figure is pasted here:
> [image: image]
> <https://user-images.githubusercontent.com/7946234/229011268-36ef9455-b2db-461e-9d77-89509fb2b9b2.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/shcaba/Sebastes_melanops_OR/discussions/17#discussioncomment-5484324>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6IZLQXK7UR7TGYTMWIKRLW6ZBULANCNFSM6AAAAAAWJP7PAM>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>


-- 

*Jason M. Cope, Ph.D.Research Fishery Biologist*
*Fishery Resource Analysis and Monitoring Division*
*Northwest Fisheries Science Center*
*2725 Montlake Blvd. East*

*Seattle, WA 98112-2013*
*NOAA Fisheries*
*Affiliate assistant faculty*
*School of Aquatic and Fishery Sciences*
*University of Washington*
***@***.*** ***@***.***>
(206) 302-2417
www.nmfs.noaa.gov

**Comment by aliwhitman** [comment]
Okay, so toggled the sectors for the GEMM report and I think I've got the final numbers for us.  I've put a file with fleet specific pivot tables in the google drive folder.  Assuming that I did it correctly, final discard rates by commercial fleet are: 

0.04 for trawl (highly variable!) 
0.009 for non-trawl (not as variable)

I just did the average annual rate, but if we want another calculation (harmonic mean?), just let me know.
